src/pymmcore_widgets/_group_preset_widget/_group_preset_table_widget.py

```diff
@@ -401,8 +401,27 @@ def _save_cfg(self) -> None:
             self, "Save Micro-Manager Configuration."
         )
         if filename:
+            if not filename.endswith(".cfg"):
+                filename += ".cfg"
             self._mmc.saveSystemConfiguration(filename)
 
+            # saveSystemConfiguration does not save the pixel size config so here
+            # we add to the saved .cfg filesave also any pixel size config.
+            if px_configs := self._mmc.getAvailablePixelSizeConfigs():
```

Comment:
it looks like this would append  a new `# PixelSize settings` section every time the file is saved. (so one might end up with a ton of those sections over time... can you confirm that?

Reply:
That's right, I didn't think about that. I think we should overwrite it...

Reply:
@tlambert03 Actually, since we create a new cfg _without_ any px info every time we call `_save_cfg`, there will always be a single `# PixelSize settings`.

Reply:
ok sounds good